### DISCUSSION TITLE
rule.reference	emergingthreats doesn't work

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#11820] [2.4] [answered]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Import

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

I7

### RAM

24GB

### Storage for /

250GB

### Storage for /nsm

?

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi all
I guess my question will be very obvious because I'm new Security Onion Student but I've been investigating for days and I can't find the solution.

I have installed it to learn, but emergingthreats links doesn't work for me and I've been days trying to know why. I'll thank any help or hint. 

For instance, in the rule.reference field appears  https://doc.emergingthreats.net/2010496  but the link doesn't work. 

I'll thank any help or hint. 

I apologize if this question is too simple, but I'm sutck here and I can't figure out a solution. 

Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
I'm checking with the Emerging Threats team to see if that resource has been moved or permanently decommissioned.

**Reply by dougburks** [reply, marked as answer]
The Emerging Threats team has confirmed that the ET rule wiki has been officially decommissioned, so I've created an issue for us to stop populating that field:
https://github.com/Security-Onion-Solutions/securityonion/issues/11846

**Comment by Carlos-mb** [comment]
¡Thank you for the fast response an solution!

Regards,
Carlos.
